### DISCUSSION TITLE
How will YOU use RSS Feeds?

### DISCUSSION THREAD

**Discussion by Moonbase59** [Moonbase59/azuracast_xmltv#6] [Announcements]
I’m thinking XMLTV Electronic Program Guides are awfully useful, but not all software understands this format. Now I think almost anything can read **RSS Feeds**—you could make your station(s) even _more_ public and also use your own RSS Feeds on your website.

Once we get more playlist info from AzuraCast (like a description), this could be _really awesome_.

It should be possible to generate
- _one_ feed _per station_, plus
- _one aggregated feed_ for _all stations_.

A first test looks promising (though far from perfect yet):

![Auswahl_294](https://github.com/Moonbase59/azuracast_xmltv/assets/3706922/d54e6c58-866c-4a46-aa24-33908c1540f7)  
_RSS Feed for one station as seen by Firefox’s "Livemarks" extension_

![Screenshot 2023-10-29 at 01-21-47 News (1225) - cloud syvi net](https://github.com/Moonbase59/azuracast_xmltv/assets/3706922/15909c92-3d82-40ef-8fdf-d9570ed0aaf0)  
_RSS Feed for one station as seen by Nextcloud_

### What do you think? Should we add this feature?

Use the **upvote button** ![Auswahl_295](https://github.com/Moonbase59/azuracast_xmltv/assets/3706922/86598820-daa4-4123-9a47-acd5069a6b9e) below this post if you don’t want to type much—thanks!

**Comment by Moonbase59** [comment]
I already introduced that feature in v0.12.0, so I’m of course interested:

- **_Do_ (or _will_) you use RSS feeds?**
- **_How_ will you use RSS feeds?**

**Reply by MauOnline** [reply]
Hey, that's a feature I'm interested in. I've been looking around to see if there was something out of the box in Azuracast. Didn't find any yet. I recently learned about podcast 2.0 specs too, which seems the best choice for support.
Basically, it would be nice to have something like what you suggested

- A feed per station
- And an aggregated feed
- The option to toggle it on/off for each station
- The feed will be automatically updated when a new episode is uploaded or the user can have the choice to add the new episode to the feed
- The new episode form will have an "RSS feed section" to add feed specific information
- I'm still reading about self-hosting the feed, but it doens't seem difficult

Happy to join any initiative for that
